src/odemis/driver/test/piezomotor_test.py

```diff
@@ -44,12 +44,14 @@
 
 KWARGS_OPEN = dict(name="test", role="test", port=PORT,
               axes={'x': {'axis_number': 1, 'speed': 0.001, 'closed_loop': False},
-                    'y': {'axis_number': 2, 'speed': 0.001, 'closed_loop': False}}
+                    'y': {'axis_number': 2, 'speed': 0.001, 'closed_loop': False}},
+                param_file="../../../../install/linux/usr/share/odemis/hwtest/pmd401.piezomotor.tsv"
```

Comment:
A bit of nitpicking, but ".piezomotor.tsv" is a little longish. How about ".pmd.tsv", it's shorter, and possibly more precise.